gui/src/app/core/Scripting/pyodide/usePyodideWorker.ts

```diff
@@ -39,6 +40,15 @@ class PyodideWorkerInterface {
       type: "module",
     });
 
+    if (window.crossOriginIsolated) {
+      this.#interruptBuffer = new Uint8Array(new SharedArrayBuffer(1));
+    } else {
+      console.warn(
+        "SharedArrayBuffer is not available, interrupting the Pyodide worker will not work",
+      );
```

Comment:
Seems like this message isn't quite right--as I understand you can still interrupt it, you're just killing the worker (and will have to reload pyodide in that case)

Reply:
Well, I guess you can no longer cancel the R scripts.

Reply:
Yes, though whether or not we even should provide this worker-killing fallback is an open question to me. What do you think?

Reply:
Eh, I don't mind it--if you're stuck in an infinite loop, or something just takes much longer than anticipated, it's nice to be able to terminate what's happening. Even if it means you have to start over next time.